nifi-nar-bundles/nifi-apicurio-bundle/nifi-apicurio-schema-registry-service/src/test/java/util/SchemaUtilsTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package util;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.nifi.apicurio.schemaregistry.util.SchemaUtils;
+import org.apache.nifi.apicurio.schemaregistry.util.SchemaUtils.ResultAttributes;
+import org.apache.nifi.schema.access.SchemaNotFoundException;
+import org.apache.nifi.serialization.record.RecordSchema;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SchemaUtilsTest {
+
+    @Test
+    void testCreateRecordSchema() throws SchemaNotFoundException, IOException {
+        final InputStream in = getResource("schema_response.json");
+
+        final RecordSchema schema = SchemaUtils.createRecordSchema(in, "schema1", 3);
+
+        assertEquals("schema1", schema.getSchemaName().get());
```

Comment:
Applies to the entire file:

Here you are using an `Optional.get()`, which can throw NPE. I don't have a strong opinion about this, especially as because this is a test code, but IMHO it would be much cleaner, if you would at least add an `.orElse(...)`, or would assert on `isPresent()` before this line.